create_infotexts.py

```diff
@@ -387,7 +368,23 @@ def process_depicted_place(self, place_string, places_mapping, desc_string):
                 self.content_cats.append(places_mapping[place_string]["commonscat"])
 
         else:
-            place_as_wikitext += place_string
+            place_matches = []
+            for place in places_mapping:
+                if place.lower() in desc_string.lower():
+                    if places_mapping[place]["wikidata"]:
```

Comment:
I repeat my question. Is it ever ok with multiple matches or does that indicate some kind of error?
In the loop I would only do `place_matches.append(place)` then only `if len(place_matches) == 1` I would add category and construct the string otherwise add the meta_cat.